### DISCUSSION TITLE
Switching between two maps

### DISCUSSION THREAD

**Discussion by ICookiee** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#272] [Q&A]
Hello,

in my Xiaomi Home app, I use two different maps. Is it possible to implement both maps?
BTW: Where can I find the card design which you're using in all the pictures? I only got the one with the colored rooms.

Thank you!

**Comment by PiotrMachowski** [comment]
Check out this issue: https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/issues/248

Colors are calculated based on room number. You can override them using `room_colors` in map extractor
